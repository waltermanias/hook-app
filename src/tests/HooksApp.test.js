import React from "react";
import { HookApp } from "../HookApp";
import { shallow } from "enzyme";

describe("HooksApp", () => {
  test("should match the snapshot", () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
