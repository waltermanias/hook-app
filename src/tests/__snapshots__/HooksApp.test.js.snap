// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HooksApp should match the snapshot 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
